libs/blocks/global-navigation/global-navigation.css

```diff
@@ -550,7 +551,11 @@ header.global-navigation {
   .feds-navItem--megaMenu .feds-popup {
     right: 0;
     justify-content: center;
-    padding: 40px 12px;
+    padding: 40px 0 0 0;
+  }
+
+  #feds-popup {
```

Comment:
As discussed on Slack, let's find a different way around higher specificity rather than using IDs. You have a bunch of options.
- A longer selector (2-3 +- a pseudo selector)
- Refactoring the CSS that leads to a longer selector to begin with to keep styles flat, you shouldn't ever need to go more than 2 or 3 selectors deep.
- Figuring out why you need to overwrite this to begin with, overwriting something as the flex direction creates more branches the code can go into & thus is harder to test/catch and might lead to more issues

Reply:
Solved 👍 Looks like the rule I was trying to override was not needed to begin with